Automattic/wp-calypso
# Update Jetpack Cloud Backup Detail Page with ActivityCardList

#### Changes proposed in this Pull Request

* Update Filterbar and ActivityCardList to hide either date range or activity type selectors
* Refactor backup detail page to hide date selector and set filter to date rang on load

#### Testing instructions

1. Navigate to `/backups`, select a site with daily backups.
2. Click "View all backup details" on a day with and without changes
3. Observe the behavior is roughly the same as production
4. boot calypso and navigate to `/`
5. Observe the filter bar on this page works as before
